### ISSUE TITLE
Additional Overgrown skybox variants

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#816]
It would be great to have the chance of choosing others custom skies for the open ceiling areas from overgrown maps, since the current white one is starting to get kinda "Monotonous"....

---

Screenshot from what i am trying to suggest:

![20160911174657_1](https://cloud.githubusercontent.com/assets/12244376/18460326/dd0cda3e-7946-11e6-8bcd-505bd789080d.jpg)

**Comment by TeamSpen210** [comment]
I do need to add that Wheatley skybox (it's seen in some of the finale maps).

**Comment by cartman1337** [comment]
I'd like to take this opportunity to ask; is there any way in the current overgrown maps to actually show the skybox? If so, what do I need to use to damage the roof, and let the sun in?

**Comment by ZombiePwner1** [comment]
This is no current way yet hammer is the only way so far.

**Comment by ellingtonvengaboysmuseum** [comment]
all he needs to do is make a custom alternate skybox to be chosen from the skybox menu... easy as cake!

**Comment by Luke18033** [comment]
It would be pretty cool to have Overgrown skyboxes for different times of day. Maybe add a night skybox and a sunrise/sunset one. For the former, items like light strips would probably need to have their brightness adjusted to make sure chambers don't become completely dark.

**Comment by Konclan** [comment]
#3891

**Comment by Luke18033** [comment]
Not really, this is referring to additional 2D skybox variants like a night/sunset sky.
